src/TN93.cpp

```diff
@@ -323,14 +323,15 @@ int main(int argc, const char *argv[]) {
   */
 
   bool cross_comparison_only = (args.input2 && !do_fst);
+  unsigned long system_CPU_count = omp_get_max_threads();
```

Comment:
You can actually delete this line, because `system_CPU_count` is left over from my debugging statements.